commands/web/phpstan

```diff
@@ -16,4 +16,15 @@ test -e phpstan.neon || curl -OL https://git.drupalcode.org/project/gitlab_templ
 sed -i 's/BASELINE_PLACEHOLDER/phpstan-baseline.neon/g' phpstan.neon
 # Add an empty baseline file to ensure it exists.
 test -e phpstan-baseline.neon || touch phpstan-baseline.neon
-phpstan analyse $DDEV_DOCROOT/modules/custom "$@"
+
+CONTRIB_MODULE_DIR=$(find "$DDEV_DOCROOT/modules/custom" -mindepth 1 -maxdepth 1 -type d | head -n 1)
+if [ -n "$CONTRIB_MODULE_DIR" ]; then
+  cd "$CONTRIB_MODULE_DIR"
+  # Ensure PHPStan configuration is symlinked from project root.
+  ln -s ../../../../phpstan.neon 2>/dev/null || true
```

Comment:
This is creating a symlink from 4 dirs above to the current dir?

Reply:
Yes, it meant to create a relative link from the repo root to the contrib module however, however, now that DRUPAL_PROJECTS_PATH was introduced which allows changing the install, I will try a different approach.